<launch>
    <node pkg="state" type="state_estimator" name="state_estimator_192_168_10_243" output="screen">
      <!-- set a private parameter for the node -->
      <param name="ip" value="192_168_10_243" />

      <!-- Sensors available 0: no 1: yes-->
      <param name="MCPTAM" value="0" />
      <param name="IMU" value="1" />
      <param name="SICK" value="0" />
      <param name="COMPASS" value="0" />
      <param name="SONAIRS" value="0" />

      <!-- All the relative pose to the centroid of the cube in the cube frame -->
      <param name="relative_pose_CM" type="string" value="$(find state)/calibration/tryphon_CM.yaml" />
      <param name="relative_pose_camera" type="string" value="$(find state)/calibration/tryphon_camera.yaml" />
      <param name="relative_pose_IMU" type="string" value="$(find state)/calibration/tryphon_IMU.yaml" />

      <!-- Pose of the SICK frame in MCPTAM frame -->
      <param name="relative_pose_SICK" type="string" value="$(find state)/calibration/tryphon_SICK.yaml" />


    </node>
    <node pkg="links" type="link_imubuff" name="link_imubuff_192_168_10_243" output="screen">
      <!-- set a private parameter for the node -->
      <param name="ip" value="192_168_10_243" />
    </node>
</launch>


